README.md

```diff
@@ -576,14 +576,14 @@ Note that by default, the `test-conformance.sh` script will run the conformance
 
 Releases are handled automatically via CI. To create a new release:
 
-1. Update the pbandk version number in `gradle.properties`, `README.md`, and `examples/*/build.gradle.kts` to remove the `SNAPSHOT` suffix. For example, if the current version is `0.9.0-SNAPSHOT`, then update it to be `0.9.0`.
-1. Commit the change. E.g.: `git commit -m "Bump to 0.9.0" -a`.
-1. Tag the new version. E.g.: `git tag -a v0.9.0`. Include the release notes for this version in the tag's description. See one of the previous tags for examples.
-1. Increment the pbandk version number in the files from step 1 to the next minor release and add the `SNAPSHOT` suffix. For example, if the tag was created for version `0.9.0`, then the new version should be `0.9.1-SNAPSHOT`.
-1. Commit the change. E.g.: `git commit -m "Bump to 0.9.1-SNAPSHOT" -a`.
+1. Update the pbandk version number in `gradle.properties`, `README.md`, and `examples/*/build.gradle.kts` to remove the `SNAPSHOT` suffix. For example, if the current version is `0.10.0-ALPHA`, then update it to be `0.10.0`.
```

Comment:
Leave the release instructions with "SNAPSHOT" instead of "ALPHA". That's the version suffix that should normally be in place on `master`. Updating these instructions from "SNAPSHOT" to "ALPHA" is just going to confuse things. Since the versions here are just meant as examples, you don't even need to update them from 0.9.0 to 0.10.0, though you could if you really wanted to.